QuoteEngine.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import yaml
+import random
+
+
+class QuoteEngine:
+
+    def __init__(self, logger):
+        self.quote_url = "https://cdn.rawgit.com/tulul/tulul-quotes/master/quote.yaml"  # noqa
```

Comment:
This is very minor. I prefer single quotes for strings (that doesn't contain other single quotes) since it's one fewer keystroke. PEP 8 has no rule for this but it says (emphasis is me) "*Pick a rule and stick to it*. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability." So I propose single quotes for string whenever possible.